feature-detects/es6/symbol.js

```diff
@@ -0,0 +1,34 @@
+/*!
+{
+  "name": "ES6 Symbol",
+  "property": "es6symbol",
+  "notes": [{
+    "name": "Official ECMAScript 6 specification",
+    "href": "https://www.ecma-international.org/ecma-262/6.0/#sec-symbol-constructor"
+  }],
+  "polyfills": ["es6shim"],
+  "authors": ["Ron Waldon (@jokeyrhyme)"],
+  "tags": ["es6"]
+}
```

Comment:
you can also add a caniuse tag here : https://caniuse.com/#feat=mdn-javascript_builtins_symbol_for

Reply:
I'm not sure where to look for these tags, all other es6 features' tags are just "es6". Maybe ["es6","symbol"] is ok ?

Reply:
changed to ["es6","symbol"]